### PR TITLE
If a single matching file fails, continue looping over remaining files rather than erroring

### PR DESCRIPTION
**Pull request by fabianvf**
Signed-off-by: Fabian von Feilitzsch <fabian@fabianism.us>

### CODE REVIEW

File: provider/builtin/provider.go [hunk 1]
```diff
@@ -142,9 +142,14 @@ func (p *builtinProvider) Evaluate(cap string, conditionInfo []byte) (lib.Provid
...
 			doc, err := xmlquery.Parse(f)
+			if err != nil {
+				// TODO(fabianvf) - do we want to error, log or continue here?
+				continue
+			}
 			list, err := xmlquery.QueryAll(doc, query)
 			if err != nil {
-				return response, err
+				// TODO(fabianvf) - do we want to error, log or continue here?
+				continue
```
Comment (shawn-hurley) [inline provider/builtin/provider.go:152]: Can we log the error as well? probably pretty high verbosity but will help IMO

Comment (shawn-hurley) [reply provider/builtin/provider.go:152]: Can we add an example of this failing to demo A for integration testing?

Comment (fabianvf) [reply provider/builtin/provider.go:152]: I can add a log. I can add a test too though it was failing for me on the `rule-example.yaml` already, did you see that behavior as well or no?

Comment (fabianvf) [reply provider/builtin/provider.go:152]: I have no idea why this is failing for me tbh

**Pull request closed without merging**